content/docs/ethernet-v0.5/fpga2.md

```diff
@@ -0,0 +1,489 @@
++++
+Description = ""
+date = "2017-04-14T13:00:00+00:00"
+title = "Running the pre-built SD-image on the FPGA"
+parent = "/docs/ethernet-v0.5/"
+prev = "/docs/ethernet-v0.5/walkthrough/"
+showdisqus = true
+
++++
+
+In this step, we want to test the Ethernet functionality on an FPGA board.
```

Comment:
This is the same description as the previous step (fpga.md). Perhaps something like "In this step we want to test booting from SD card, as well as Ethernet support"?